index.html

```diff
@@ -46,5 +42,189 @@ <h3 class="logo">John Doe </h3>
     <div class="bckimg">
         <img class="sideimg"src="images/illustration-header-mobile.png" alt="illustration">
     </div>
+    
+     <!--Second Branch: Recent Work and About me Page-->
+<section class="workSection">
+    <div>
+    <p class="workSecPara">My Recent Works</p>
+    
+    <img class="workSectionRec" src="images/rec2.png" alt="Rec2">
+</div>
+
+<div>
+    <img class="workSectionImg"src="images/Placeholder.png" alt="Section Image">
+    <p class="wrkSecHeader">Multi-Post Stories</p>
```

Comment:
- I believe this should be in an `<hx>` tag as well. Just as you did with the `<h4>Profesional Art Printing Data</h4>` 🌷